lib/member_email_decorator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'scraped'
+require 'cgi'
+require 'execjs'
+require 'nokogiri'
+
+class MemberEmailDecorator < Scraped::Response::Decorator
```

Comment:
as a general rule, I think we always want basic documentation for Decorators. The names are rarely quite enough to know what they're doing at a glance, and although this one isn't too difficult to follow, it still requires reading the code to see, for example, whether it's actually _replacing_ the text, or adding the decoded address as an extra attribute.